docs/development/history.md

```diff
@@ -0,0 +1,2 @@
+```{include} ../../CHANGELOG.md
```

Comment:
This changelog is for testing purpose, not sure it's worth it to include it here.
Or did you plan to tweak the test and have a proper one for the project ?

Reply:
You've read my mind ;)